tests/ttnn/multichip_unit_tests/test_multidevice_TG.py

```diff
@@ -196,6 +196,7 @@ def test_galaxy_matmul_2d_fracture(M, K, N, weights_dtype, cluster_shape, device
     assert out_pass
 
 
+@pytest.mark.skip("See GH #10673: DRAM-SHARDED Matmuls gives ND PCC on TG")
```

Comment:
You mentioned `main` has some ND pcc issue on TG. Was this the only source of the issue ?

Reply:
TG llama mlp was also seeing same issues due to using DRAM-sharded matmuls